### PR TITLE
feat: push job history to upstream

### PR DESCRIPTION
**Pull request by yashmehrotra**
Fixes: https://github.com/flanksource/mission-control/issues/701

### CODE REVIEW

File: schema/system.hcl [hunk 1]
```diff
@@ -198,6 +203,11 @@ table "job_history" {
     type    = timestamptz
     default = sql("now()")
   }
+  column "is_pushed" {
```
Comment (moshloop) [inline schema/system.hcl:206]: I think we need an is_pushed index

Comment (yashmehrotra) [reply schema/system.hcl:206]: Added 👍🏽

File: schema/system.hcl [hunk 2]
```diff
@@ -198,7 +203,16 @@ table "job_history" {
...
+    null    = false
+    default = false
+    type    = bool
+  }
   primary_key {
     columns = [column.id]
   }
+  index "job_history_is_pushed_idx" {
+    columns = [column.is_pushed]
+    where   = "is_pushed IS FALSE"
```
Comment (moshloop) [inline schema/system.hcl:216]: This should match the where clause above

Comment (yashmehrotra) [reply schema/system.hcl]: My bad, fixed it

**Pull request merged**